example/accel/offload-template/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+#------------------------------- -*- cmake -*- -------------------------------#
+# Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+#-----------------------------------------------------------------------------#
+# Minimal Geant4 application with Celeritas offloading capabilities
+#-----------------------------------------------------------------------------#
+cmake_minimum_required(VERSION 3.10)
+cmake_policy(VERSION 3.12...3.30)
+project(offload-template)
+
+# Export/set Celeritas_ROOT environment variable pointing to its install dir
+# Defaults to celeritas/install
+set(ENV{Celeritas_ROOT} "../../../../install")
+
```

Comment:
It might be good to delete this custom bit of code (and this probably shouldn't be set through the environment anyway). If you want to make it user friendly, you could drop the REQUIRED on 17/18 and then print a helpful message (pointing to `Celeritas_ROOT` or `CMAKE_INSTALL_PREFIX`).

Reply:
Thanks. I wasn't sure how to make this easy/automatic, and the message solution is much better indeed.